pkg/oracle/filter.go

```diff
@@ -0,0 +1,40 @@
+package oracle
+
+import (
+	"encoding/json"
+
+	//"k8s.io/client-go/util/jsonpath"
+	"github.com/PaesslerAG/jsonpath"
+)
+
+func filter(value, path string) ([]byte, error) {
+	var v interface{}
+	if err := json.Unmarshal([]byte(value), &v); err != nil {
+		return nil, err
+	}
+	result, err := jsonpath.Get(path, v)
+	if err != nil {
+		return nil, err
+	}
+	return json.Marshal([]interface{}{result})
```

Comment:
Why an array wrapper?

Reply:
Because the library we use returns single result `Get`, C# node needs an array.

Reply:
OK.